website/docs/Debugging-Scripts.md

```diff
@@ -7,7 +7,7 @@ title: Debugging Scripts
 Since Premake's update to 5.3, the only debugger that seems to be able to debug Premake is the free ZeroBrane Studio IDE.
```

Comment:
Unrelated, but we should update wording like this. It reads more like a community review, than a statement from the development team. @starkos